src/Analyzers/CSharp/Analyzers/UseImplicitObjectCreation/CSharpUseImplicitObjectCreationDiagnosticAnalyzer.cs

```diff
@@ -87,7 +87,7 @@ private void AnalyzeSyntax(SyntaxNodeAnalysisContext context)
                     ConversionOperatorDeclarationSyntax conversion => conversion.Type,
                     OperatorDeclarationSyntax op => op.ReturnType,
                     BasePropertyDeclarationSyntax property => property.Type,
-                    AccessorDeclarationSyntax { Parent: BasePropertyDeclarationSyntax baseProperty } accessor when accessor.IsKind(SyntaxKind.GetAccessorDeclaration) => baseProperty.Type,
+                    AccessorDeclarationSyntax { Parent: AccessorListSyntax { Parent: BasePropertyDeclarationSyntax baseProperty } } accessor when accessor.IsKind(SyntaxKind.GetAccessorDeclaration) => baseProperty.Type,
```

Comment:
I think this would work as well:

```suggestion
                    AccessorDeclarationSyntax { RawKind: (int)SyntaxKind.GetAccessorDeclaration, Parent: AccessorListSyntax { Parent: BasePropertyDeclarationSyntax baseProperty } } => baseProperty.Type,
```

Reply:
sounds good.